cspell.yaml

```diff
@@ -68,6 +68,7 @@ words:
   - golangci
   - gomnd
   - gomock
+  - goodtotal
```

Comment:
you shouldn't need to add this if you didn't actually misspell it here :)
![image](https://github.com/nobl9/nobl9-go/assets/48822818/3f18e2a5-48e7-495b-b2c1-7c1bc640b2c6)

Reply:
I actually want it to be called this way, but I've reworded the test case name to avoid adding this word to cspell.